policies/rules.md

```diff
@@ -146,6 +146,8 @@ To participate, team members must be at least 13 years of ago at the time the co
 
 Prizes amounts and types will be determined at the discretion of organizers and event sponsors, and distributed in a manner they see fit.
 
+In the event of a tie for a prize awarded by vote, then he winner will be deemed the first project to receive the complete set of votes. If this does not resolve the tie then the organizers reserve the right to determine the winner by a manner deemed appropriate and by the organizers judgement to be most fair and impartial.
```

Comment:
> then he

`the`

---

> complete set of votes

I liked our initial wording of "the final vote", "complete set" is vague to me.

---

>If this does not resolve the tie then the organizers reserve the right to determine the winner by a manner deemed appropriate and by the organizers judgement to be most fair and impartial.

If this does not resolve the tie, the organizers reserve the right to determine a winner based on their judgment.  The organizers will strive to do so in an appropriate, fair and impartial manner.

Reply:
Good call @ajyong. Just making both of these changes now. Thanks!